app/models/user.rb

```diff
@@ -327,7 +327,7 @@ def react_to_role_change
   end
 
   def consents_to_processing
-    unless processing_consent
+    if processing_consent=="0"
```

Comment:
Would this assume consent if the `processing_consent` param was missing entirely?

Reply:
Yes.
I've added a presence validation in the model.